scripts/job_manager.ps1

```diff
@@ -0,0 +1,143 @@
+<#
+.SYNOPSIS
+
+class to ease job management
+Adds a job to a "Tracker"; a Tracker is a set of similar jobs that should be
```

Comment:
You can add a PowerShell Job to a Tracker. A Tracker is a label for a set of jobs that you want to manage in the same way (start, stop, wait).